tests/testthat/test-structural-properties.R

```diff
@@ -729,3 +729,21 @@ test_that("coreness() works", {
   gc <- coreness(g)
   expect_equal(gc, c(3, 3, 3, 2, 2, 2, 2, 2, 2, 2))
```

Comment:
https://github.com/r-lib/testthat/issues/1669 is the reason why I renamed the scripts.